002-storage.md

```diff
@@ -0,0 +1,186 @@
+# Storage fleet
+
+# **Storage fleet**
+
+- When database is smaller then a chunk size we naturally can store them in one chunk (since their page_key would fit in some chunk's [hi, lo) range).
+
+<img width="937" alt="Screenshot_2021-02-22_at_16 49 17" src="https://user-images.githubusercontent.com/284219/108729836-ffcbd200-753b-11eb-9412-db802ec30021.png">
+
+Few databases are stored in one chunk, replicated three times
```

Comment:
Is this for simplicity? Do we want to consider availability zones? A failure of two machines would make the database unavailable.

Reply:
Each chunk is stored on N nodes, so failure of one would not affect availability. I was suggesting to store different databases in one chunks in order to simplify management of case with many small databases.